### PR TITLE
🚀 Add Notion.so login feature to get token

### PR DESCRIPTION
**Pull request by NamamiShanker**
## Related Issue
- Get token from Notion.so login
- We still need to find a way to upload playbook content to Notion.so using this token.

## Describe the changes you've made
Added `settings.py` file to contain global variables and to set file paths. Added user login feature, which will allow you to get token from user login through the browser. 

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

## Screenshots

notion

### CODE REVIEW

**Comment by GouravSardana** [comment]
Add some screenshots as well, If possible

**Comment by GouravSardana** [comment]
@guptaaastha @pranavbaburaj You guys can also share your thoughts, If you want to change anything from your end or any improvement.

**Comment by NamamiShanker** [comment]
I'm so tired of codacy

**Comment by NamamiShanker** [comment]
> Add some screenshots as well, If possible

Added a GIF of working. Please help me fix this codacy issue.

**Comment by pranavbaburaj** [comment]
I have resolved the codacy issues 🙂

**Review by pranavbaburaj — APPROVED** [review]
It looks good to me 🙂

**Comment by NamamiShanker** [comment]
> I have resolved the codacy issues

Thank you so much!

**Comment by GouravSardana** [comment]
After logged in. Does it say in ther terminal that login success or failure ?

**Comment by NamamiShanker** [comment]
> After logged in. Does it say in the terminal that login success or failure ?

I'll add this

**Comment by NamamiShanker** [comment]
@pranavbaburaj @GouravSardana Codacy keeps conflicting itself with Docstring styling. I have currently submitted the code in standard PEP8 notation with Sphinx rEST doc styling. Please see to the conflicts. I have also added Login Success and Failure messages.

**Review by guptaaastha — CHANGES_REQUESTED** [review]
@NamamiShanker everything looks fine, just add a login status' as follows
- Successfully logged into Notion 🎉 
- Unsuccessfull login. Please try again 😕 
(use emojis in the messages to be consistent and user friendly 😄 )

Also put a small explanation in the help message as to how to use this feature.
Adding a description in the documentation will also be helpful

**Comment by NamamiShanker** [comment]
@guptaaastha I have made some changes. Added emojis in messages. Added entry in the documentation and put details in the help message.

**Comment by NamamiShanker** [comment]
@pranavbaburaj @GouravSardana What do you guys suggest? Do I try to fix these Codacy warnings because it's conflicting with itself right now? Or do you guys prefer some changes?

**Comment by pranavbaburaj** [comment]
2021-05-27-16-52-github com
Hey @NamamiShanker, your branch is 12 commits behind the main branch. I recommend you to fetch and marge 🙂

**Comment by GouravSardana** [comment]
Everything LGTM. @pranavbaburaj can you check the codacy issue with him or else I'll check it later today

**Comment by pranavbaburaj** [comment]
> Everything LGTM. @pranavbaburaj can you check the codacy issue with him or else I'll check it later today

Sorry, I didn't understand what the issue is

**Comment by NamamiShanker** [comment]
> Hey @NamamiShanker, your branch is 12 commits behind the main branch. I recommend you to fetch and marge.

@pranavbaburaj Yep! done.

**Review by pranavbaburaj — APPROVED** [review]
Looks good 👍🏾

**Review by anshsrtv — APPROVED** [review]
Great job! @NamamiShanker 🚀

File: src/arguments/error.py [hunk 1]
```diff
@@ -18,3 +18,19 @@ def evoke_search_error(self, error_statement):
         ]
         for text_to_print in print_text:
             print(text_to_print)
+
+class LoginError():
+    def __init__(self, error_statement, success=False):
+        """Implements error printing for User Login
```
Comment (anshsrtv) [inline src/arguments/error.py:24]:
```suggestion
        """
        Implements error printing for User Login
```

File: src/arguments/notion.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        return driver.get_cookies()
+    except Exception as e:
+        print(e)
+    finally:
+        driver.quit()
+
+class NotionClient():
+
+    """
+
```
Comment (anshsrtv) [inline src/arguments/notion.py:45]:
```suggestion
```

**Review by guptaaastha — APPROVED** [review]
LGTM 
good work @NamamiShanker !

**Comment by guptaaastha** [comment]
@NamamiShanker @pranavbaburaj Can you look into Codacy issues?
Checks are failing

**Comment by pranavbaburaj** [comment]
> @NamamiShanker & @pranavbaburaj Can you look into Codacy issues?
> Checks are failing

Sorry @guptaaastha, I couldn't find the issues with codacy 😑

**Comment by NamamiShanker** [comment]
> @NamamiShanker & @pranavbaburaj Can you look into Codacy issues?
> Checks are failing

Me neither. Don't understand why is it failing.

**Comment by guptaaastha** [comment]
@GouravSardana @pranavbaburaj @NamamiShanker 
This thread seems related to the Codacy issue we're facing (D203) [https://github.com/PyCQA/pydocstyle/issues/141](https://github.com/PyCQA/pydocstyle/issues/141)
Have a look

**Pull request merged**